bower.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "KaTeX",
-  "version": "0.8.0-pre",
+  "name": "katex",
   "main": [
-    "dist/katex.min.js",
-    "dist/katex.min.css"
+    "dist/katex.js",
+    "dist/katex.css"
```

Comment:
I couldn't find any documentation about *why* that requirement exists. bower/bower@1c62adcdb6f09875434de2d886c23731a0bae53c just says it's following the spec, and https://github.com/bower/spec/blob/master/json.md had that line since the content got copied from Google docs.

I can only guess that they want subsequent steps to minify the sources. The way I understand it, they might even handle modules, so specifying our main unbrowserified entry point in `bower.json` *might* work, too.